FizzBuzz/Program.cs

```diff
@@ -9,10 +10,11 @@ static void Main(string[] args)
             Run();
         }
 
+        /* Using a string to store output instead of write command so only one write is needed and no extra boolean var */
```

Comment:
Is this comment really useful? You should only need to add comments if it's something someone will need to know when they're reading the code for the first time. And you should always consider whether it's possible to refactor the code to make it more readable instead of adding a comment. Ideally your code should be so neat and clear that no comments are necessary 🙂